lib/hours.php

```diff
@@ -1029,7 +1029,7 @@ function locationNameToIdOriginal( $name ) {
 function title_filter( $where, &$wp_query ) {
 	global $wpdb;
 	if ( $search_term = $wp_query->get( 'search_prod_title' ) ) {
-		$where .= ' AND ' . $wpdb->posts . '.post_title LIKE \'%' . esc_sql( like_escape( $search_term ) ) . '%\'';
+		$where .= ' AND ' . $wpdb->posts . '.post_title LIKE \'%' . esc_sql( $wpdb->esc_like( $search_term ) ) . '%\'';
```

Comment:
This change is most likely not needed, in that the file as a whole is unused (this was part of the old hours system that can probably just be removed)